lib/usd/ui/layerEditor/resources.qrc

```diff
@@ -0,0 +1,34 @@
+<RCC>
+    <qresource prefix="/UsdLayerEditor">
+
+        <file alias="LE_save_all_100.png">resources/save_all_100.png</file>
```

Comment:
Is there a reason to use the "alias" tag. In Maya we only put the alias on the _100 version to name it without the _100

Reply:
it's because the alias removes the subdirectory name